README.md

```diff
@@ -1,4 +1,7 @@
-![Build Status](http://storage.googleapis.com/java-runtimes-kokoro-build-badges/openjdk-runtime-master.png)
+![Build Status](http://storage.googleapis.com/java-runtimes-kokoro-build-badges/openjdk-runtime-master.png) 
+
+![Stability](https://img.shields.io/badge/gcr.io/google--appengine/openjdk:8-stable-green.svg)
+![Stability](https://img.shields.io/badge/gcr.io/google--appengine/openjdk:9-unstable-red.svg)
 
 # Google Cloud Platform OpenJDK Docker Image
 
```

Comment:
`runtime: java` is still correct. The user selects between openjdk8/9 using `runtime_config.jdk` (see below). Until we release the openjdk:9 image and add it to the runtime builder, `jdk: openjdk9` won't be available. I'll update the readme to reflect the current state.

```yaml
runtime_config:
  jdk: openjdk8
```